### PR TITLE
docs: `/docs/concepts/features-overview/`

### PR DESCRIPTION
**Pull request by Yohe-Am**
- Re-does the feature overview page of the documentation.

## Summary by CodeRabbit

- **New Features**
  - Added a "Features Roadmap" component to the website, displaying a list of features with details and links.

- **Documentation**
  - Updated various guides and reference documents to improve clarity and presentation of code examples.
  - Added new sections for various features such as Typegate, Typegraph, Runtimes, Prisma, Auth, Tooling, and SDK.

- **Bug Fixes**
  - Corrected a typo in the GraphQL runtimes reference documentation.

- **Refactor**
  - Replaced `SDKTabs` and `TabItem` components with `TGExample` for better code example presentation.
  - Adjusted the `MiniQL` component to handle optional properties and default settings.

### CODE REVIEW

**Comment by Yohe-Am** [comment]
Sample screenshot:

<details>
image
</details>

File: website/docs/concepts/features-overview/index.mdx [hunk 1]
```diff
@@ -2,54 +2,801 @@
...
+        title: "Cron triggers",
+        status: "future",
+        link: "/docs/reference/typegate",
+        body: (
+          <>
+            Execute materializers at certain time intervals.
+            Useful for cleanup jobs.
+          </>
+        ),
+      },
```
Comment (Yohe-Am) [inline website/docs/concepts/features-overview/index.mdx:69]: TODO: future feature for graphql subscriptions.

**Review by afmika — DISMISSED** [review]

File: website/docs/concepts/features-overview/index.mdx [hunk 2]
```diff
@@ -2,54 +2,814 @@
...
+          </>
+        ),
+      }, 
+    ],
+  ]}
+/>
+
+### Prisma
+
+Access a myriad of databases directly from your typegraph. Run queries, mange it's schema, the full orm featureset.
```
[the remark this answers is not in the document]
Comment (afmika) [reply website/docs/concepts/features-overview/index.mdx:426]: Also `mange`  => `manage`

**Review by zifeo — APPROVED** [review]
Great structure and work!!

File: website/docs/reference/runtimes/graphql/index.mdx [hunk 3]
```diff
@@ -15,12 +15,12 @@ Update `typegraph.py` with the highlighted lines below:
...
+Again, a few interesting things happened here:
 
 1. No migration has been run. The field `user` comes from another runtime and doesn't exist in the database. The typegate will orchestrate the query execution in all runtimes and minimize the work done.
-2. The `from_parent` rule automatically fills the input type with the parent field named `uid`. The `g(·)` rule allows making named reference to another type and avoid circular reference.
+2. The `from_parent` rule automatically fills the input type with the parent field named `uid`. The `g(·)` rule allows making named references to another type and avoids circular references.
 
-Other type enforcement rules also exists:
+Other type enforcement rules also exist:
 
 - `from_secret(key)` to fill the input type with the secret in the `TG_[typegraph name]_[key]` format
```
Comment (Natoandro) [inline website/docs/reference/runtimes/graphql/index.mdx:25]: I guess this is obsolete now...

**Review by Natoandro — COMMENTED** [review]
Great 👍

Added some comment... Not really relatés to your changes...

**Review by destifo — APPROVED** [review]

**Review by destifo — APPROVED** [review]
🔥

**Pull request merged**